ADAL/src/cache/ADTokenCacheAccessor.m

```diff
@@ -282,7 +283,13 @@ - (void)updateCacheToItem:(ADTokenCacheItem *)cacheItem
     NSString* savedRefreshToken = cacheItem.refreshToken;
     if (isMRRT)
     {
-        AD_LOG_VERBOSE(correlationId, @"Token cache store - Storing multi-resource refresh token for authority: %@", _authority);
+        if ([ADAuthorityUtils isKnownHost:_authority])
+        {
+          AD_LOG_VERBOSE(correlationId, @"Token cache store - Storing multi-resource refresh token for authority: %@", _authority);
+        } else {
+            AD_LOG_VERBOSE(correlationId, @"Token cache store - Storing multi-resource refresh token");
+            AD_LOG_VERBOSE_PII(correlationId, @"Token cache store - Storing multi-resource refresh token for authority: %@", _authority);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Split this 137 characters long line (which is greater than 120 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=objc%3ALineLength)